requirements/edx-sandbox/base.txt

```diff
@@ -4,6 +4,7 @@
 #
 #    make upgrade
 #
+
```

Comment:
I just filed [TE-2593](https://openedx.atlassian.net/browse/TE-2593) for this nuisance.  Could you make sure we don't switch the whitespace needlessly in this PR before that gets fixed?

Reply:
Sure thing